x/bank/keeper/genesis.go

```diff
@@ -20,10 +20,6 @@ func (k BaseKeeper) InitGenesis(ctx sdk.Context, genState types.GenesisState) {
 			panic(err)
 		}
 
-		if err := k.ValidateBalance(ctx, addr); err != nil {
-			panic(err)
-		}
-
```

Comment:
Is this safe to remove? May be we still need `ValidateBalance` after `SetBalances` below? @alexanderbez any thoughts?

Reply:
According to my investigation - yes, because we are calling `k.ClearBalances(ctx, addr)` in `k.SetBalances`.

Reply:
Yes, I  think we should do it in a separate post, because this will be another breaking feature. Also, not sure if that will land in release,  @clevinson, @aaronc  , what do you think?

Reply:
Please also have a look here: https://github.com/cosmos/cosmos-sdk/issues/7682#issuecomment-718318758